Library/Formula/spacenavd.rb

```diff
@@ -0,0 +1,47 @@
+class Spacenavd < Formula
+  desc "Userspace device driver/daemon for 3Dconnexion's 3D input devices."
+  homepage "http://spacenav.sourceforge.net/index.html"
+  url "https://downloads.sourceforge.net/project/spacenav/spacenav%20daemon/spacenavd%200.6/spacenavd-0.6.tar.gz"
+  sha256 "c2d203bf96c5a959590146a43fe5d6e5e8c5c38a8b2f55aa199d967d0d88d0ab"
+
+  option "with-x11", "Enable support for x11 communication."
+
+  depends_on :x11 => :optional
+
+  patch :DATA
+
+  def install
+    args = %W[
+      --disable-debug
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+    ]
+
+    args << "--disable-x11" if build.without? "x11"
+
+    system "./configure", *args
+
+    system "make", "install"
+  end
+
+  test do
+    system "spacenavd", "-h"
```

Comment:
Sorry for the delay, I just got done with a pretty busy weekend.

The problem with changing the file locations is that the client library used to communicate with the daemon also has these locations hard-coded.  I could modify the `libspnav` package as well as this one to change the paths, but anyone that is using the client library outside of homebrew (many packages just include the source in their repo) won't be connect to the homebrew-installed instance of `spacenavd`.  Thoughts?